tokio-postgres/src/config.rs

```diff
@@ -34,6 +34,8 @@ pub enum TargetSessionAttrs {
     Any,
     /// The session must allow writes.
     ReadWrite,
+    /// The session allow only reads.
+    ReadOnly,
```

Comment:
This presumably needs to be used somewhere in the connection code.

Reply:
Sry, haven't committed my latest changes. Please have a look.